### DISCUSSION TITLE
license for Nager.Date

### DISCUSSION THREAD

**Discussion by corleymanz** [nager/Nager.Date#478] [General]
Hello,

I want to use Nager.Date in my software for offline access. The documentation states it needs a local support requires a license key. But the software is open source under MIT  licence.

my question is : "can i freely use Nager.Date without pay the licence cost?"

(the license key is in the binary and can be acquired)

**Comment by tinohager** [comment]
No one can forbid you to compile and use the software yourself. It's up to you to calculate what your own time costs and if it's important for you to support the projects you use so they can be kept up to date.
